recording_server/src/media_context.ts

```
@@ -1,22 +1,100 @@
 import { ChildProcess, spawn } from 'child_process'
 import internal from 'stream'
+import { platform } from 'os'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)